lang/po/Firmware_sk.po

```diff
@@ -458,7 +459,7 @@ msgstr ""
 #. MSG_BTN_CONTINUE c=5
 #: ../../Firmware/mmu2/errors_list.h:282 ../../Firmware/mmu2/errors_list.h:292
 msgid "Done"
-msgstr ""
+msgstr "Hotov"
```

Comment:
@3d-gussner there is no space for c=6 at least? In SK language `Done=Hotovo`

Reply:
I have found the `Done` being used with two other options with 4 events and we have to keep it limited to 5 char. Sorry.

Reply:
OK thank you.. in that case, it can stay as `Hotov`